lib/rules/require-to-use-local-storage-utils.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @fileoverview Disallow to use localStorage directly
+ * @author Jan Rochalski<joonny220@gmail.com>
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Disallow to use localStorage directly',
+      category: 'Stylistic Issues',
+      recommended: false,
+    },
+    fixable: null, // or "code" or "whitespace"
+    schema: [
+      // fill in your schema
+    ],
+  },
+
+  create: function (context) {
+    return {
+      VariableDeclarator(node) {
```

Comment:
I believe this rule does not work for usages without assigning to variables:

<img width="944" alt="image" src="https://github.com/user-attachments/assets/0db46cd7-f224-433c-9dfc-3f3d75c813fd">